.github/workflows/publish_nugets.yml

```diff
@@ -0,0 +1,56 @@
+# v1.0.0
+name: Module nuget publishing
+
+on:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    if: ${{ github.actor != 'dependabot[bot]' && (github.event.pull_request.head.repo.full_name == github.repository || github.event.pull_request.head.repo.full_name == '') }}  # Check that PR not from forked repo and not from Dependabot
+    runs-on: ubuntu-latest
+    env:
+      CLIENT_SECRET: ${{secrets.CLIENT_SECRET}}
+      SONAR_TOKEN: ${{secrets.SONAR_TOKEN}}
+      GITHUB_TOKEN: ${{ secrets.REPO_TOKEN }}
+      NUGET_KEY: ${{ secrets.NUGET_KEY }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install VirtoCommerce.GlobalTool
+        uses: VirtoCommerce/vc-github-actions/setup-vcbuild@master

```

Comment:
change to 
`run: dotnet tool install --global VirtoCommerce.GlobalTool --version 2.1.0-alpha.17`
Latest vc-build not working with .net6